openshift/template.yaml

```diff
@@ -87,7 +87,7 @@ objects:
           - name: API_BACKBONE_SERVICE_PORT
             value: "5000"
           - name: API_BACKBONE_SERVICE_TIMEOUT
-            value: "120"
+            value: "300"
```

Comment:
Let's use a variable here so that it can be handled through saas-analytics PR directly